drivers/uktty/pl011/Config.uk

```diff
@@ -13,4 +14,13 @@ config LIBUKTTY_PL011_EARLY_CONSOLE_BASE
 	default 0
 
 endif
+
+config LIBUKTTY_PL011_OUTPUT
```

Comment:
Given that we have one early console device, I think these essentially overlap with the `Early Console` option. Same for `ns16550`.